srcpkgs/pnpm/template

```diff
@@ -0,0 +1,35 @@
+# Template file for 'pnpm'
+pkgname=pnpm
+version=5.8.0
+revision=1
+# We won't use wrksrc, but it's needed to silence "cannot access wrksrc directory" error
+create_wrksrc=yes
+build_style=meta
```

Comment:
Please use build_style=fetch , and remove skip_extraction. build_style=meta would allow package to be empty.

Reply:
Changed to fetch and removed skip_extraction.

Should I remove `create_wrksrc=yes`? It seems not needed anymore. The error is no longer there.

Reply:
I've removed it, since there was no issue without it.